utils/_context/_scenarios.py

```diff
@@ -1145,6 +1145,14 @@ class scenarios:
         doc="",
     )
 
+    debugger_method_probes_snapshot = EndToEndScenario(
+        "DEBUGGER_METHOD_PROBES_SNAPSHOT",
+        proxy_state={"mock_remote_config_backend": "DEBUGGER_METHOD_PROBES_SNAPSHOT"},
+        weblog_env={"DD_DYNAMIC_INSTRUMENTATION_ENABLED": "1", "DD_REMOTE_CONFIG_ENABLED": "true",},
+        library_interface_timeout=10,
+        doc="",
```

Comment:
Could you add a quick `doc="bla bla bla"` explaining the purpose of this scenario ?

Reply:
added here 688137364e0832fc0c5d3ad9d7f6ceae43b283f9